cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -44,6 +44,14 @@ const (
 	defaultContainerdConfigFilePath = "/etc/containerd/config.toml"
 	defaultCrioConfigFilePath       = "/etc/crio/crio.conf"
 	defaultDockerConfigFilePath     = "/etc/docker/daemon.json"
+
+	defaultConfigSource = configSourceCommand
+	configSourceCommand = "command"
+	configSourceFile    = "file"
+
+	runtimeContainerd = "containerd"
+	runtimeCrio       = "crio"
+	runtimeDocker     = "docker"
```

Comment:
Nit: these are the same as the `nvidia-toolkit` values. Is there a common place that we could put them? Happy to do this as a refactor though.

Reply:
We could consider package-level consts in the `engine` package: `engine.Docker`, `engine.Containerd`, `engine.Crio`?

I'm happy to look into this as a follow-up though.